uint16 front
uint16 front_right
uint16 front_left
uint16 back_right
uint16 back_left
uint16 back 
